vpr/src/place/move_utils.h

```diff
@@ -324,4 +328,9 @@ bool intersect_range_limit_with_floorplan_constraints(t_logical_block_type_ptr t
 
 std::string e_move_result_to_string(e_move_result move_outcome);
 
+#ifndef VTR_ENABLE_DEBUG_LOGGING
```

Comment:
1. Should be #ifdef, not $ifndef
2. Needs a comment saying what the routine does.